delfin/leader_election/distributor/task_distributor.py

```diff
@@ -47,7 +48,9 @@ def __call__(self):
                       six.text_type(e))
 
     def distribute_new_job(self, task_id):
-        executor = CONF.host
+        partitioner = ConsistentHashing()
+        partitioner.start()
+        executor = partitioner.belong_to_host(task_id)
```

Comment:
partitioner.belong_to_host returns a byte literal , we need to convert to string  may be with 
 executor = partitioner.belong_to_host(task_id).decode('utf-8')

Reply:
ok, thanks